src/workspaces/WorkspacePolicies/WorkspacePolicies.test.tsx

```diff
@@ -138,35 +138,45 @@ describe('WorkspacePolicies', () => {
       render(<WorkspacePolicies />);
 
       // Assert
-      expect(screen.queryByText(policyLabel)).toBeNull();
+      expect(screen.queryByText(policyTitle)).toBeNull();
     });
   });
 
-  describe('supports UI customization ', () => {
-    it('allows passing a title and label about the list', async () => {
+  describe('displays a help link about security/policies', () => {
+    const policyLinkText = 'Learn more about Terra security';
+    const azurePolicyHref = '21329019108635-Host-FISMA-data-on-FedRAMP-moderate-Terra-Azure';
+
+    it('renders a link with Azure security information for Azure billing projects', async () => {
       // Act
-      render(
-        <WorkspacePolicies title="Test title" policiesLabel="About this list:" workspace={protectedAzureWorkspace} />
-      );
+      render(<WorkspacePolicies billingProject={azureProtectedDataBillingProject} />);
 
       // Assert
-      expect(screen.getAllByText('Test title')).not.toBeNull();
-      expect(screen.getAllByText('About this list:')).not.toBeNull();
+      const link = screen.getByRole('link', { name: policyLinkText });
+      expect(link.getAttribute('href')).toContain(azurePolicyHref);
     });
 
-    it('renders a link if provided', async () => {
+    it('renders a link with Azure security information for Azure workspaces', async () => {
       // Act
-      render(
-        <WorkspacePolicies
-          workspace={protectedAzureWorkspace}
-          policiesLink={<Link href="http://dummy-link">about policies</Link>}
-        />
-      );
+      render(<WorkspacePolicies workspace={protectedAzureWorkspace} />);
 
       // Assert
-      screen.getByText('about policies');
+      const link = screen.getByRole('link', { name: policyLinkText });
+      expect(link.getAttribute('href')).toContain(azurePolicyHref);
     });
 
+    it('renders a link with auth domain information for Gcp protected workspaces', async () => {
+      // Act
+      render(<WorkspacePolicies workspace={protectedGoogleWorkspace} />);
+
+      // Assert
+      const link = screen.getByRole('link', { name: policyLinkText });
+      expect(link.getAttribute('href')).toContain(
+        '360026775691-Overview-Managing-access-to-controlled-data-with-Authorization-Domains'
+      );
```

Comment:
Nit: perhaps slightly more idiomatic (and possibly a clearer error message) as:

```suggestion
      expect(link).toHaveAttribute('href', expect.stringContaining(
        '360026775691-Overview-Managing-access-to-controlled-data-with-Authorization-Domains'
      ));
```

Reply:
Hmmm, I actually find the way it currently is to be much clearer (as far as code, that is). The `expect.stringContaining` is pretty confusing to me as an argument to the `toHaveAttribute` call.